Cargo.toml

```diff
@@ -44,21 +44,19 @@ axum = { version = "0.5.17", optional = true, features = ["http2"] }
 axum-server = { version = "0.5.1", optional = true, features = ["rustls", "rustls-pemfile", "tls-rustls"] }
 base64 = { version = "0.21.2", optional = true }
 bitvec = "1.0"
-# TODO: the only place where we use bytes is in ByteArrStream. ordering_mpsc is a replacement for it
-# so we can remove this dependency after migrating to it. It is added only to avoid having `web-app` feature enabled
-# by default.
 bytes = "1.4"
 clap = { version = "4.0", optional = true, features = ["derive"] }
 comfy-table = { version = "7.0", optional = true }
 config = "0.13.2"
 criterion = { version = "0.5.1", optional = true, default-features = false, features = ["async_tokio", "plotters", "html_reports"] }
 dashmap = "5.4"
+derivative = "2.2"
```

Comment:
is it worth it and does it make our compilation time significantly worse or no?

Reply:
I have removed derivative in the latest commit.